### ISSUE TITLE
Release date of the installed version

### ISSUE THREAD

**Issue by benispeti** [BePo65/license-report#193]
Is it possible to add a "Build in" field support for the date of the installed version? 
(I can see that the date of the remote version is already supported, but that's not what I'm looking for)

**Comment by BePo65** [comment]
which field in the package.json or the npmjs package data contains the `date of the installed version` (whatever that might be)?

**Comment by benispeti** [comment]
As far as I see this information is not available from package.json. I found the following way to gather the date of a particular npm package, not sure if this helps.
`npm view rxjs time[6.6.6]`

**Comment by BePo65** [comment]
This won't be easy. By now we can only define fields from the package.json in our config file.

What do you use the date for? To check when the package was updated?

**Comment by BePo65** [comment]
As you probably know we have a field `latestRemoteModified` that contains the date for the latest version on the server. Will this be enough?

**Comment by benispeti** [comment]
Yes, I saw the `latestRemoteModified` field, that was not what I was looking for. Btw how the latest remote version and modified date is queried?

**Comment by BePo65** [comment]
The `latestRemoteModified` is set in the `addPackageDataFromRepository.js` file (here named `lastModified`).

As you know, we get the 'remote' data from the registry (e.g. for [semver](https://registry.npmjs.org/semver).

For me it remains the question, what you will use these dates for? To get a feeling, how old the version is compared to the latest available?

**Comment by benispeti** [comment]
I checked the [semver](https://registry.npmjs.org/semver) example and I think the date information is available for each release version too, not only the lastModified date. Copying a snippet here:
```
"time": {
    "created": "2011-02-12T00:20:25.690Z",
    "modified": "2024-07-31T15:24:46.517Z",
    "1.0.0": "2011-02-12T00:20:26.037Z",
    "1.0.1": "2011-02-18T17:15:49.775Z",
    "1.0.2": "2011-03-22T21:27:35.218Z",
    ...
```

Based on the registry response I think the release date of a given version could be identified.

**Comment by BePo65** [comment]
i know that the date is available, but the question is: "what do you need this date for" and "why should it be added to the tool" (i.e. is it a use case "many" other could have too?).

So would you be so kind, to tell me a little bit more about your use case?

**Comment by BePo65** [comment]
closed issue because of inactivity for 5 months
